cinema/serializers.py

```diff
@@ -1 +1,84 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Actor,
+    Genre,
+    CinemaHall,
+    Movie,
+    MovieSession
+)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ("first_name", "last_name", "full_name")
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
```

Comment:
sometimes you use `all`, sometimes field names, try to keep your project one-styled